Simplenote/Classes/SPSearchController.swift

```diff
@@ -190,6 +223,28 @@ extension SPSearchController: UISearchBarDelegate {
 
     func searchBarCancelButtonClicked(_ searchBar: UISearchBar) {
         dismiss()
-        delegate?.searchControllerDidEndSearch(self)
+    }
+}
+
+
+// MARK: - SPSearchBar
+//
+class SPSearchBar: UISearchBar {
+
+    /// **Custom** Behavior:
+    /// Normally resigning FirstResponder status implies all of the button subviews (ie. cancel button) to become disabled. This implies that
+    /// hiding the keyboard makes it impossible to simply tap `Cancel` to exit **Search Mode**.
+    ///
+    /// With this (relatively safe) workaround, we're keeping any UIButton subview(s)  enabled, so that you can just exit Search Mode anytime.
+    ///
+    @discardableResult
+    override func resignFirstResponder() -> Bool {
```

Comment:
I wish there was another way but I can't think of anything better than this. 
This would be a good candidate for a UI test to catch if the behavior changes in a future iOS release.

Reply:
Added a note in the main [Tech Debt issue](https://github.com/Automattic/simplenote-ios/issues/481). Thank you sir!!